include/vara/Feature/FeatureSourceRange.h

```diff
@@ -71,15 +71,18 @@ class FeatureSourceRange {
   FeatureSourceRange(fs::path Path,
                      std::optional<FeatureSourceLocation> Start = std::nullopt,
                      std::optional<FeatureSourceLocation> End = std::nullopt,
-                     Category CategoryKind = Category::necessary)
+                     Category CategoryKind = Category::necessary,
+                     std::optional<std::string> MemberOffset = std::nullopt)
       : Path(std::move(Path)), Start(std::move(Start)), End(std::move(End)),
-        CategoryKind(CategoryKind) {}
+        CategoryKind(CategoryKind), MemberOffset(std::move(MemberOffset)) {}
 
   FeatureSourceRange(fs::path Path, FeatureSourceLocation Start,
                      FeatureSourceLocation End,
-                     Category CategoryKind = Category::necessary)
+                     Category CategoryKind = Category::necessary,
+                     std::string MemberOffset = "")
```

Comment:
Why do we need two different `MemberOffset` defaults? Having `nullopt` and `""`. Would it not be better to have the same behavior independent on which ctor is called?